Example/.flowconfig

```diff
@@ -1,58 +1,58 @@
 [ignore]
-
-# We fork some components by platform.
+; We fork some components by platform
 .*/*[.]android.js
 
-# Ignore templates with `@flow` in header
-.*/local-cli/generator.*
-
-# Ignore malformed json
-.*/node_modules/y18n/test/.*\.json
-
-# Ignore the website subdir
-<PROJECT_ROOT>/website/.*
-
-# Ignore BUCK generated dirs
+; Ignore "BUCK" generated dirs
 <PROJECT_ROOT>/\.buckd/
 
-# Ignore unexpected extra @providesModule
-.*/node_modules/commoner/test/source/widget/share.js
+; Ignore unexpected extra "@providesModule"
+.*/node_modules/.*/node_modules/fbjs/.*
+.*/react-native-tag-input/node_modules/fbjs/.*
 
-# Ignore duplicate module providers
-# For RN Apps installed via npm, "Libraries" folder is inside node_modules/react-native but in the source repo it is in the root
+; Ignore duplicate module providers
+; For RN Apps installed via npm, "Libraries" folder is inside
+; "node_modules/react-native" but in the source repo it is in the root
 .*/Libraries/react-native/React.js
 .*/Libraries/react-native/ReactNative.js
-.*/node_modules/jest-runtime/build/__tests__/.*
+
+; Ignore parent directory's react-native
+.*/react-native-tag-input/node_modules/react-native/.*
+
+; Ignore any duplicate symlinked version of ourselves
+.*/node_modules/react-native-tag-input/Example/.*
```

Comment:
This is avoid getting ourselves symlinked back, which would lead to literally every module having a duplicate